deploy/examples/operatorsource.cr.yaml

```diff
@@ -1,11 +1,11 @@
 apiVersion: marketplace.redhat.com/v1alpha1
 kind: OperatorSource
 metadata:
-  name: community-operators
-  namespace: openshift-marketplace
+  name: upstream-community-operators
+  namespace: marketplace
```

Comment:
Should this be `openshift-marketplace`?

Reply:
The section starts with 
> The following section assumes that Marketplace was installed in the openshift-marketplace namespace

I suppose it makes more sense to replace all instances of `openshift-marketplace` with just `marketplace`